tree/tree/src/TBranchElement.cxx

```diff
@@ -5239,6 +5248,7 @@ void TBranchElement::SetAddress(void* addr)
    // We do this only once because it depends only on
    // the type of our object, not on its address.
    if (!fInitOffsets) {
```

Comment:
Is this check thread-safe?

Reply:
No, but it does not need to.  Only one thread should call SetAddress on the same branch object at a time.